ibis/backends/clickhouse/registry.py

```diff
@@ -512,6 +513,17 @@ def _string_like(translator, expr):
     )
 
 
+def _group_concat(translator, expr):
+    arg, sep, where = expr.op().args
+    if where is not None:
+        arg = where.ifelse(arg, ibis.NA)
+        arg = translator.translate(arg)
```

Comment:
Is this needed? You're already applying `translator.translate` to `arg` in the `return`.

Do you mind adding another test that hits the code in the `if` please.

Reply:
you are right，it will cause a bug 😣